setup.py

```diff
@@ -5,12 +5,12 @@
 
```

Comment:
I am not sure if the following is needed for people who want to install from source (meaning `git clone`, etc.):
```suggestion
with open("docs/requirements.txt", "r") as fr:
    requirements = fr.read().splitlines()
```
and then...
```python
    setup(... etc
    install_requires=requirements
```

Reply:
I'm not entirely sure if we should. First, by adding this to `setup.py` it adds the `sphinx` libraries and dependencies for any install. If the intent is to just use the code by doing `python setup.py install` then you don't need sphinx, etc. We always prefer to be on the side of simpler build. I think there is a way to do this more elegantly, but not entirely sure. For example, tests dependencies can be specified via the `tests_require` argument. But there's no option for `setuptools`. See: https://github.com/pypa/setuptools/issues/1820

Note that the `sphinx` CI builds uses the `docs/` folder, which is why there is a separate requirements.txt there. It was the simplest way to ensure the `sphinx` builds pass.